src/main/java/http/routes/EchoBody.java

```diff
@@ -0,0 +1,26 @@
+package http.routes;
+
+import http.Codes;
+import http.Route;
+
+import java.util.List;
+
+public class EchoBody implements Route {
+    final String CRLF;
+    final List<String> headers;
+    final String headersResponse;
+
+    public EchoBody() {
+        CRLF = "\r\n";
+        headers = List.of("POST");
+        headersResponse = String.format("Allow: %s", headers.get(0));
+    }
+
+    @Override
+    public String response(String method, String body) {
```

Comment:
refactor: this would be a nice place to introduce the [builder pattern](https://refactoring.guru/design-patterns/builder). 

For example: This `response` method returns a `Response` object instead of a string, 
```
 @Override
    public Response response(String method, String body) {
        if (headers.contains(method)) {
        return new ResponseBuilder()
                .withStatusCode(Codes.HTTP_VERSION.getCode())
                .withHeader(headersResponse)
                .withBodyByte(body.getBytes())
                .build();
        }
 if (headers.contains(method)) {
        return new ResponseBuilder()
                .withStatusCode(Codes._405.getCode())
                .withHeader(headersResponse)
                .withBodyByte(body.getBytes())
                .build();
    }
```
    
   The `ResponseBuilder` class will contain methods that create all the diff components of the response, as well as a `build()` method that concatenates the response:
 
```
public class ResponseBuilder {
     public ResponseBuilder withStatusCode(StatusCode statusCode) {
          // does something
      }
      public Response build() {
        return new Response(buildStatusLine(), buildHeaders(), buildBodyByte());
    }
   }
```
   
   A `Response` class which contains getter/setter classes for all the parts of a response.

Reply:
Have added a version of the builder pattern which implements a `ResponseBuilder` and `Response` classes.